### DISCUSSION TITLE
Run a web search from command line

### DISCUSSION THREAD

**Discussion by trdsr** [browsh-org/browsh#458] [Ideas]
e.g. when I ask myself how 'tar' works and I want to look it up in the web, I could run
`browsh -s "tar extract files"` to see the search results as the first page browsh renders in my terminal.

**Comment by tombh** [comment]
This is possible with the HTTP server mode described here in the docs: https://www.brow.sh/docs/in-browser-usage

Eg: `curl --header "X-Browsh-Raw-Mode: PLAIN" localhost:4333/https://www.google.com/search?q="tar extract files"`
